spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfig.scala

```diff
@@ -76,6 +78,8 @@ case class StandardizationConfig(rawFormat: String = "xml",
   override def withPerformanceMetricsFile(value: Option[String]): StandardizationConfig = copy(performanceMetricsFile = value)
   override def withFolderPrefix(value: Option[String]): StandardizationConfig = copy(folderPrefix = value)
   override def withPersistStorageLevel(value: Option[StorageLevel]): StandardizationConfig = copy(persistStorageLevel = value)
+  override def withFixedWidthTreatEmptyValuesAsNulls(value: Option[Boolean]): StandardizationConfig = copy(fixedWidthTreatEmptyValuesAsNulls = value)
```

Comment:
**Minor:** Line too long

Reply:
This is a good place for the exception we allowed ourselves. We said on CQC that a line shouldn't be longer than 120. But if it is justifiable it is ok to go to 140. Now, remember I fought for numbers 80 and 100. But here I think it is justifiable to have it longer since it does not have any "logic". It is just a `copy` method adding that one parameter, 20 times repeating, for different parameters.

Reply:
I take it back, I was on a wrong branch and didn't see this one goes to 150... 🤦

Reply:
Also in such case it's fine to add a scalastyle ignore rule.